Hashing/DoubleHashing.java

```diff
@@ -0,0 +1,76 @@
+import java.util.Scanner;
+class HashMap{
+    public int size;
+    public int elements;
+    public int[] table;
+    public int collision;
+    HashMap(){
+        System.out.println("Enter the size of the hash table: ");
+        Scanner sc = new Scanner(System.in);
+        size = sc.nextInt();
+        table = new int[size];
+        for(int i=0;i<size;i++){
+            table[i] = -1;
+        }
+        elements = 0;
+    }
+    boolean isFull(){
+        return elements == size;
+    }
+    int hashFunction(int key){
+        return key%size;
+    }
+    int hasFunction2(int key){
+        return 7-(key%7);
```

Comment:
Why would you use 7 here ,can we use any other number ? explain about it.

Reply:
Yes, I can use prime number which tends to generate a unique hash every time.
I have used hardcoded `7` (prime number). I can write a get prime function to get a prime number every time.